CHANGELOG.md

```diff
@@ -7,39 +7,50 @@ Platform support legends:
     * Note: the papercuts will majorly be with the build process. Things might be very rough to integrate as no polish at all has been given yet.
 * ❌ = tier 3 support. It doesn't work just yet, but we plan to make it work.
 
-## [0.5.0] - 2022-14-09
+## [0.5.1] - 2022-21-09
 
 <details>
     <summary>git-conventional changelog</summary>
 
 ### Bug Fixes
 
-- Rollback openmls & chrono in order to release 0.5.0
```

Comment:
This is weird, it's not supposed to overwrite the past log, see my other comment